Extension/src/main.ts

```diff
@@ -279,6 +284,11 @@ async function postInstall(info: PlatformInformation): Promise<void> {
 }
 
 async function finalizeExtensionActivation(): Promise<void> {
+    if (vscode.workspace.getConfiguration("C_Cpp", null).get<boolean>("disableLanguageService")) {
```

Comment:
It seems like this feature should be a little more involved than just this.  What happens to all the commands that are registered in package.json?  Are we able to remove the context menu items?

Reply:
Yeah, all the package.json and context menu stuff remains activated, but they don't do anything. I was thinking that didn't matter, and/or wait till people complain about that. I don't think there's a way to remove the package.json stuff without modifying the file and doing a Reload Window and/or splitting the extension into multiple ones and disabling the separate sub-extension.